### PR TITLE
[Add] Add a new set of websites

### PR DESCRIPTION
**Pull request by Bhavesh0327**

### CODE REVIEW

**Comment by sdmg15** [comment]
Hi @Bhavesh0327 you'll need to remove duplicates links from your Pull Request.
Only one or two links per pull request is appreciable

**Comment by Bhavesh0327** [comment]
Yes @sdmg15 sir for sure, can you please let me know which links should i remove?  I added 3 links actually, and i felt like all three were important and supposed to be there
i. leetcode
ii. Aditya verma Youtube channel
iii. Rust coding style guide

File: README.md [hunk 1]
```diff
@@ -653,6 +654,7 @@ When learning CS, there are some useful sites you must know to get always inform
 </div>
 
 ## Video Tutorials
+- [Aditya Verma](https://www.youtube.com/channel/UC5WO7o71wvxMxEtLRkPhiQQ) : Algorithm tutorials playlists by an Indian youtuber Aditya verma.
```
Comment (sdmg15) [inline README.md:657]: The colon (`:`) has an extra space

Comment (Bhavesh0327) [reply README.md]: Okay I changed that too.

**Pull request merged**